### ISSUE TITLE
0.9.0 broken, cannot find directory ./lib ?

### ISSUE THREAD

**Issue by greglearns** [gobblejs/gobble#54]
gobble: error starting gobble
>>>
Cannot find module './lib'
Error: Cannot find module './lib'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (./node_modules/gobble/index.js:2:18)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
<<<

**Comment by Rich-Harris** [comment]
Woah, that's really weird. It's working for me... (the least satisfying answer to any problem, ever!). Could you try doing a `rm -rf node_modules/gobble` and reinstalling to see if the problem persists? Failing that, is there anything unusual about your project setup?

**Comment by greglearns** [comment]
lol, no worries about "it's working for me."

```rm -rf node_modules/gobble``` solved the problem. Before the problem occurred, I did ```npm update``` so I wonder if things ended up in a weird state.

Thank you!

One strange thing, is it intended that ```gobble build <dirname> -f``` never exits after it is called? Previously, in 0.7.14, gobble exited, but now it does not. (Also, now I get an "error" about the ractive.js.map file... not sure what that is about)

```
~/D/l/k/u/f/ui git:develop ❯❯❯ gobble build build -f                                                                                                                                         ⏎ ⬆ ✚ ✖ ✱ ◼
node.map() is deprecated. You should use node.transform() instead for both file and directory transforms
GOBBLE INFO    { code: 'BUILD_START' }
GOBBLE INFO    { code: 'TRANSFORM_START',
  progressIndicator: true,
  id: '01-moveTo' }
GOBBLE INFO    { code: 'TRANSFORM_COMPLETE', id: '01-moveTo', duration: 5 }
GOBBLE INFO    { code: 'TRANSFORM_START',
  progressIndicator: true,
  id: '02-ractive' }
GOBBLE INFO    { code: 'TRANSFORM_COMPLETE', id: '02-ractive', duration: 40 }
GOBBLE INFO    { code: 'TRANSFORM_START',
  progressIndicator: true,
  id: '03-browserify' }
GOBBLE INFO    { code: 'TRANSFORM_COMPLETE',
  id: '03-browserify',
  duration: 1241 }
GOBBLE INFO    { code: 'TRANSFORM_START',
  progressIndicator: true,
  id: '04-moveTo' }
GOBBLE INFO    { code: 'TRANSFORM_COMPLETE', id: '04-moveTo', duration: 1 }
GOBBLE INFO    { code: 'TRANSFORM_START',
  progressIndicator: true,
  id: '05-moveTo' }
GOBBLE INFO    { code: 'TRANSFORM_COMPLETE', id: '05-moveTo', duration: 1 }
GOBBLE INFO    { code: 'MERGE_START', id: '06-merge', progressIndicator: true }
GOBBLE INFO    { code: 'MERGE_COMPLETE', id: '06-merge', duration: 8 }

GOBBLE ERROR   ENOENT, open '(truncated)/.gobble/06-merge/1/home/ractive.js.map'
============
stack trace:
Error: ENOENT, open '(truncated)/.gobble/06-merge/1/home/ractive.js.map'
============


GOBBLE INFO    Successfully built project to build
(... hangs, does not exit)
```

**Comment by Rich-Harris** [comment]
Good news, we got to the bottom of this issue, via https://github.com/gobblejs/gobble/issues/55. Will put out a new release soon.

**Comment by Rich-Harris** [comment]
Just released gobble 0.9.1 alongside gobble-cli 0.4.0 - between them, this should be fixed now. Let me know if not!

**Comment by greglearns** [comment]
works for me. Thanks!
